monailabel/tasks/infer/basic_infer.py

```diff
@@ -267,6 +267,9 @@ def __call__(
         req = copy.deepcopy(self._config)
         req.update(request)
 
+        # model options
+        self.path.append(os.path.join(os.path.dirname(self.path[0]), req.get("models", "model.pt")))
```

Comment:
Can we use model_name or model_filename as the key?

Reply:
Used model_filename